teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(name, age, _parent_permission, specialization)
+    super(name, age, parent_permission: true)
```

Comment:
- [ ] Based on feedback on the person class above these two lines should be updated as well.

  ```
  def initialize(age, specialization, name = 'unknown', parent_permission: parent_permission)
    super(age, name, parent_permission: parent_permission)
      ...
  ```